reactor-core/src/main/java/reactor/core/publisher/DirectProcessor.java

```diff
@@ -79,7 +79,8 @@
  * </p>
  *
  * @param <T> the input and output value type
- * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+ * @deprecated To be removed in 3.5, prefer clear cut usage of {@link Sinks}:
+ * <pre>Sinks.many().multicast().onBackpressureError();</pre> for example.
```

Comment:
remove the `for example`, `onBackpressureError()` is the one and only way to get a `DirectProcessor`.